lib/ddtrace/contrib/rails/patcher.rb

```diff
@@ -6,6 +6,14 @@ module Rails
       # Patcher
       module Patcher
         include Base
+
+        DEFAULT_INSTRUMENTATIONS = {
+          action_view_rendering: true,
+          action_controller_processing: true,
+          active_support_caching: true,
+          active_record: true
```

Comment:
Just a thought, albeit requiring no change at all: this `active_record` line is mean to enable/disable "sub-integrations" that the Rails meta-integration activates.

For what it's worth, I had started on changes to address exactly this kind of configuration setting in my [configuration overhaul](https://github.com/DataDog/dd-trace-rb/compare/feature/integration_configuration_settings...refactor/active_record_integration_configuration?expand=1), which ultimately was meant to produce a new configuration API that resembles [this prototype](https://gist.github.com/delner/b55092afa1650a614b9d17e2416a282f). The purpose of this is eerily similar to that.